#resolution of the lidar sensor
resolution: 0.05
# topic name for lidar scan to subscribe to
topic_name: /scan_front
# angle for proper orientation 
pallet_orientation_angle: -172
# intensity of lidar rays when they strike the aruco marker or the docker/charger
pallet_intensity: 100
# error value wrt marker intensity i.e
# marker intensity +- marker intensity threshold 
makrer_intensity_threshold: 25
# length of the aruco marker / object in use for docking which will form a clluster in the point cloud
cluster_length: 0.1
# error value wrt cluster length i.e
# cluster length +- cluster_length_threshold
cluster_length_threshold: 0.03
#max permissible error in slope of the marker from the required slope of the marker in the charger.yaml file
max_delta_error_slope: 2
#width of the robot
robot_width: 0.177
#length of the robot
robot_length: 0.3
# distance of obstacle at which the robot comes to a halt
object_distance_threshold: 0.2
#minimum number of points within a given thresold distance to qualify a point as 
min_points_obstacle: 3
#max error limit outside which the revovery will start
#i.e if err_y > 0.3 -> start recovery which is to go to the docking position
max_err_y: 0.3
#kp parameter
kp: 0.05
#kd parameter
kd: 0.08
#time to wait fpr obstacle clearance s	before aborting the task
time_for_obstacle_clearance: 10
cluster_depth_threshold: 0.2
# length of pallet in m
pallet_length: 2
kp_orientation: 0.001
kd_orientation: 0.005
kp_movement: 0.8
kd_movement: 0.1
max_linear_speed: 0.1
max_angular_speed: 0.1


